engine/src/main/java/org/terasology/rendering/assets/shader/ShaderProgramFeature.java

```diff
@@ -21,18 +21,12 @@
 /**
  */
 public enum ShaderProgramFeature {
-    FEATURE_REFRACTIVE_PASS
-            (0b00000001),
-    FEATURE_ALPHA_REJECT
-            (0b00000010),
-    FEATURE_LIGHT_POINT
-            (0b00000100),
-    FEATURE_LIGHT_DIRECTIONAL
-            (0b00001000),
-    FEATURE_USE_MATRIX_STACK
-            (0b00100000),
-    FEATURE_USE_FORWARD_LIGHTING
-            (0b01000000);
+    FEATURE_REFRACTIVE_PASS (0b00000001),
+    FEATURE_ALPHA_REJECT (0b00000010),
+    FEATURE_LIGHT_POINT (0b00000100),
+    FEATURE_LIGHT_DIRECTIONAL (0b00001000),
+    FEATURE_USE_MATRIX_STACK (0b00100000),
+    FEATURE_USE_FORWARD_LIGHTING (0b01000000);
```

Comment:
Historically we've avoided it. Probably from some sort of "hard to maintain" or keep consistent angle, maybe in part from being unable to write style rules that can consider lined up elements (may end up getting complaints about excessive spaces)